src/input/element-input.js

```diff
@@ -867,7 +904,7 @@ class ElementInput {
             var element = this._elements[i];
 
             if (! element.screen || ! element.screen.screen.screenSpace) {
-                if (this._checkElement(rayA, element, false)) {
+                if (this._checkElement(rayA, element, false) > 0) {
```

Comment:
If failure is defined as `value < 0` then success should be `value >= 0`.

(Make sure it's consistent throughout).